_docs-v3/toolbar/titleRangeSeparator.md

```diff
@@ -0,0 +1,34 @@
+---
+title: titleRangeSeparator
+---
+
+The date range in the **agendaWeek** title is, by default, separated by " - " between the beginning and end date.
+
+`titleRangeSeparator` allows the user to override and customize the range separator (ex. changing the separator from " - " to " to "). The user's customized separator will be added to the default range format, as well as the customized range format generated with `titleFormat`.
+
+```
```

Comment:
for block code samples like this (aka code samples that have three backticks), please start them fully aligned to the left. in other words, this code sample is indented too much